components/empty-state/empty-state-illustrated-link.js

```diff
@@ -0,0 +1,57 @@
+import { html, LitElement, nothing } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { EmptyStateIllustratedMixin } from './empty-state-illustrated-mixin.js';
+import { linkStyles } from '../link/link.js';
+import { runAsync } from '../../directives/run-async/run-async.js';
+import { styleMap } from 'lit/directives/style-map.js';
+
+/**
+ * The `d2l-empty-state-illustrated-link` component is an empty state component that displays an illustration and action link. The illustration property can be set to use one of the preset illustrations or a custom SVG illustration can be added in the default slot.
+ * @slot - Custom SVG content if `illustration-name` property is not set
+ */
+class EmptyStateIllustratedLink extends EmptyStateIllustratedMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			  * Optional: The action URL or URL fragment of the link
+			  * @type {string}
+			  */
+			actionHref: { type: String, attribute: 'action-href' },
+		};
+	}
+
+	static get styles() {
+		return [super.styles, linkStyles];
+	}
+
+	constructor() {
+		super();
+		this._illustratedComponentType = 'link';
```

Comment:
Hmm it is possible. To do that, I imagine we would need to add a unique ID to each component that we could reference. There doesn't seem to be any precedence for this in our other components which just reference the component type but it could be valuable for us to start doing this.

Reply:
Ah sorry I was more thinking we could just go the other way and simplify the error messages to not include type of link or button

Reply:
May not be necessary, but I figured that this was not a lot of overhead and would allow more specific error messages. It allows the error messages to denote the exact component name which could make it easier to find when searching for it rather than something like `Illustrated empty state component is missing x` and then requiring the consumer to search and find out what the name of the component is.